### ISSUE TITLE
Daemon: throw error  `unexpected end of file` when upload a plugin from tarball.

### ISSUE THREAD

**Issue by FeelyChau** [alibaba/pipcook#445]
Here is the logs:
```shell
Run npm run test:pipeline -- text-bayes-classification
  npm run test:pipeline -- text-bayes-classification
  shell: /bin/bash -e {0}

> pipcook@ test:pipeline /Users/runner/work/pipcook/pipcook
> sh ./run_pipeline.sh "text-bayes-classification"

[start]: run pipeline from ./test/pipelines/text-bayes-classification.json
[start]: installing plugin ./packages/plugins/data-collect/csv-data-collect
[start]: installing @pipcook/plugins-csv-data-collect from /Users/runner/work/pipcook/pipcook/packages/plugins/data-collect/csv-data-collect
[fail]: unexpected end of file
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! pipcook@ test:pipeline: `sh ./run_pipeline.sh "text-bayes-classification"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the pipcook@ test:pipeline script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/runner/.npm/_logs/2020-08-13T04_52_47_092Z-debug.log
##[error]Process completed with exit code 1.
```
We need more logs to fix this.
#444

**Comment by yorkie** [comment]
Is this fixed by #457?

**Comment by FeelyChau** [comment]
> Is this fixed by #457?

Fixed.
